### ISSUE TITLE
> @BaseMax Can you answer some of my questions regarding this repo?

### ISSUE THREAD

**Issue by jalotra** [serverok/squid-proxy-installer#6]
> @BaseMax Can you answer some of my questions regarding this repo?

Hello @jalotra,

What's your question? (Create a new issue and ask there...)

Regards,
Max

_Originally posted by @BaseMax in https://github.com/serverok/squid-proxy-installer/pull/4#issuecomment-522263895_

**Comment by jalotra** [comment]
@BaseMax This repository has 3 .conf files, namely squid.conf, squid-ip.conf, squid-2.conf. I am not able to whitelist some of the domains that I have. I have used squid-2.conf file and changed the ip.allow files with my custom domains that I wanted to whitelist. 
Line 15, Squid-2.conf states making ip.allow file. `acl hostonnet_ips src "/etc/squid/ip.allow"`.
What have I done wrong?

**Comment by BaseMax** [comment]
> @BaseMax This repository has 3 .conf files, namely squid.conf, squid-ip.conf, squid-2.conf.

Only "squid.conf" file has called in the bash scripts [squid3-install.sh](https://github.com/serverok/squid-proxy-installer/blob/master/squid3-install.sh), and [squid_configure_ip.sh](https://github.com/serverok/squid-proxy-installer/blob/master/squid_configure_ip.sh).

Regards,
Max Base

**Comment by jalotra** [comment]
Yup, I know that. Can you tell me what squid_configure_ip.sh does? I am not able to get what multiple ips have to do?

**Comment by BaseMax** [comment]
> I am not able to get what multiple ips have to do?

I don't know.
Do you mean `servers.txt` [file](https://github.com/serverok/squid-proxy-installer/blob/master/fabric/servers.txt)?

-----

> Can you tell me what squid_configure_ip.sh does? 

Updating the squid config.
You can check it using:

```
$ cat /etc/squid/squid.conf
```

Regards,
Max Base

**Comment by BaseMax** [comment]
You can create a file and add in a list of sites to be whitelisted `/etc/squid/sites.whitelist.txt`.

e.g: [copied from a page](http://www.webdnstools.com/articles/squid-proxy-whitelist)
```
.example.com
.google.com
.bing.com
```

Restart squid using:
```
$ squid -k reconfigure
```
Or start the squid service.

Regards,
Max Base

**Comment by jalotra** [comment]
Thankyou @BaseMax for the resources and your time. I configured my proxy server.

**Comment by BaseMax** [comment]
> Thankyou @BaseMax for the resources and your time. I configured my proxy server.

Hello,

Thank you for your reply, Enjoy. @jalotra 

Best,
Max
